llvm/test/Transforms/LoopVectorize/AArch64/clamped-trip-count.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt -S < %s -passes=loop-vectorize -mtriple aarch64-linux-gnu -mattr=+sve 2>&1 | FileCheck %s
+
+define void @clamped_tc_8(ptr nocapture %dst, i32 %n, i64 %val){
+; CHECK-LABEL: define void @clamped_tc_8
+; CHECK: call void @llvm.masked.store.nxv8i8.p0(<vscale x 8 x i8> %19, ptr %20, i32 1, <vscale x 8 x i1> %active.lane.mask)
```

Comment:
Unfortunately, I think now that you're only matching a single CHECK line and not using update_test_scripts.py you will need to make this CHECK line more robust. It refers directly to SSA variables such %19, etc. How about something like?

`CHECK: call void @llvm.masked.store.nxv8i8.p0(<vscale x 8 x i8> {{.*}}, ptr {{.*}}, i32 1, <vscale x 8 x i1> {{.*}})`

Reply:
Done